webrender_traits/src/types.rs

```diff
@@ -372,6 +372,35 @@ impl Into<f64> for SubpixelOffset {
 }
 
 #[derive(Clone, Hash, PartialEq, Eq, Debug, Deserialize, Serialize, Ord, PartialOrd)]
+pub struct SubpixelPoint {
```

Comment:
Any reason we can just do `type SubpixelPoint = euclid::Point2D<SubpixelOffset>` ?